ingest/src/main/scala/geotrellis/osme/ingest/GenerateHistogram.scala

```diff
@@ -0,0 +1,29 @@
+package geotrellis.osme.ingest
+
+import geotrellis.raster._
+import geotrellis.raster.histogram._
+import geotrellis.spark._
+import geotrellis.spark.io._
+import geotrellis.spark.io.s3._
+import geotrellis.vector._
+import geotrellis.spark.util.SparkUtils
+import org.apache.spark.SparkConf
+import org.apache.spark.rdd.RDD
+import spray.json.DefaultJsonProtocol._
+
+object GenerateHistogram {
```

Comment:
This has already been merged as part of pervious PR. This is probably what is giving you a merge conflict here.